### ISSUE TITLE
Missing accessibility labels for all the charts

### ISSUE THREAD

**Issue by Aksha93** [Abhinandan-Kushwaha/react-native-gifted-charts#1043]
### Description

Hi,

Thanks for this library. I tried working with this library and see that there is no support for accessibility labels. Are there any plans to add them?

I could not find them in the list of supported props. Can you suggest? Thanks in advance!

### Steps to reproduce

1. Add any library component like BarChart or PieChart.
2. Enable screen reader
3. Missing accessibility label

### version of react-native-gifted-charts

1.4.58

### React Native version

0.73.6

### Platforms

iOS

### Workflow

React Native

**Comment by Aksha93** [comment]
@Abhinandan-Kushwaha any update on adding accessibility labels for x-axis and yaxis labels and values?
